pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md

```diff
@@ -97,33 +107,13 @@ Follow these steps to enable Kaptain in air-gapped and networked environments fr
 
 1.  Create the resource in the workspace you just created, which instructs Kommander to deploy the `AppDeployment` to the `KommanderCluster`s in the same workspace.
 
-<p class="message--important"><strong>IMPORTANT: </strong>If you are deploying Kaptain to an attached cluster, ensure that either the <code>defaults</code> or <code>overrides</code> contain the <code>${WORKSPACE_NAMESPACE}</code> in the <code>global.workspace</code> section of the <code>values.yaml</code>. For an example of this, observe the <code>ConfigMap</code> configuration in the <a href="#enable-kaptain-with-a-custom-configuration-using-the-cli">Enable Kaptain with a custom configuration using the CLI</a> section.</p>
+## Enable Kaptain with a custom configuration using the CLI (Essential and Enterprise)
 
-## Enable Kaptain with a custom configuration using the CLI
+<p class="message--important"><strong>IMPORTANT: </strong>If you are deploying Kaptain to a managed or attached cluster, ensure that the <code>ConfigMap</code> contains the <code>${WORKSPACE_NAMESPACE}</code> in the <code>global.workspace</code> section of the <code>values.yaml</code>, as shown in the following example.</p>
 
 If you want to customize your installation and modify the custom domain name, external Dex, creation of profiles, certificates, for example, continue with these steps:
 
-1.  Provide the name of a `ConfigMap` in the `AppDeployment`, which provides custom configuration on top of the default configuration:
-
-    ```yaml
-    cat <<EOF | kubectl apply -f -
-    apiVersion: apps.kommander.d2iq.io/v1alpha2
-    kind: AppDeployment
-    metadata:
-      name: kaptain
-      namespace: ${WORKSPACE_NAMESPACE}
-    spec:
-      appRef:
-        kind: App
-        name: kaptain-2.0.0
-      configOverrides:
-        name: kaptain-overrides
-    EOF
-    ```
-
-1.  Create the `ConfigMap` with the name provided in the step above, with the custom configuration:
-
-    <p class="message--important"><strong>IMPORTANT: </strong>If you are deploying Kaptain to an attached cluster, ensure that the <code>ConfigMap</code> contains the <code>${WORKSPACE_NAMESPACE}</code> in the <code>global.workspace</code> section of the <code>values.yaml</code>, as shown in the following example.</p>
+1.  Create the `ConfigMap` with the custom configuration. In the following example, the ConfigMap is configuring Kaptain to communicate with the [Management cluster via Dex](../../configuration/external-dex/), which is a necessary step when deploying Kaptain to a Managed or Attached cluster (Enterprise only). Other configurations can be made in the same way.
```

Comment:
minor: on confluence, the word "Enterprise" in this line has a link to [this page](https://d2iq.atlassian.net/wiki/spaces/DENT/pages/29656544/DKP+Enterprise). If such a page is available on GH as well, we could add a link to it here